src/Symfony/Component/Console/Command/Command.php

```diff
@@ -328,7 +328,7 @@ public function setCode(callable $code): static
             $code = $code(...);
         }
 
-        $this->code = new InvokableCommand($this, $code);
+        $this->code = new InvokableCommand($this, $code, true);
```

Comment:
Maybe use a named argument for true value? To make it more verbose.

Reply:
We don't use named arguments in Symfony, because we don't guarantee them in [our BC promise](https://symfony.com/doc/current/contributing/code/bc.html#note-10). In this case, it doesn't matter, since the classes are in the same package. But for me it's not useful, my IDE displays the name of the argument in the function call.